orc8r/tools/docker/install_gateway.sh

```diff
@@ -13,13 +13,15 @@
 
 # This script is intended to install a docker-based gateway deployment
 
+# Both ENV vars are moved in to .env file:
+# GIT_HASH="v1.3.3"
```

Comment:
Does GIT_HASH not contain a hash value? Is it more of a release tag name?

Reply:
This env var is used later in the script as 
` git -C $INSTALL_DIR/magma checkout "$GIT_HASH"`
so it's basically anything you can git checkout :
- commit ID
- tag
- branch